projects/sd-jwt-dotnet.md

```diff
@@ -0,0 +1,68 @@
+# Project Name
+
+SD-JWT-DotNet
+
+# Preferred Maturity Level
+
+Labs
+
+# Project Description
+
+A Dot Net implementation of the [Selective Disclosure for JWTs](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-04.html) (SD-JWT) specification.
+
+# Alignment with the OpenWallet Foundation Mission
+
+This library implements the Selective Disclosure for JWTs specification, which is an important building block for future digital identity wallets.
+
+# Code of Conduct
+
+None
+
+# TAC Sponsor
+
+None
+
+# Project License
+
+Apache 2.0.
+
+# Source Control
+
+[https://github.com/thomas-tran/sd-jwt-dotnet](https://github.com/thomas-tran/sd-jwt-dotnet)
```

Comment:
Before we will be able to transfer this repo to the OpenWallet Foundation, we will need to do one of the options found here: https://github.com/openwallet-foundation/project-proposals/#fixing-dco-on-existing-repositories

Reply:
Done

Reply:
It looks like only the last three have been signed off. Could you make sure to sign-off all previous commits too?

Reply:
Done, please let me know if I missed anything else